ApollosProject/apollos-embeds
# chore: (web) Change content single sibling items to be 2 column on large screens.

## 🐛 Issue

[(web) Change content single sibling items to be 2 column on large screens.](https://3.basecamp.com/3926363/buckets/27088350/card_tables/cards/6849485410)

## ✏️ Solution

Remove 3 columns for large screens, leave it at 2 columns max

## 🔬 To Test

1. navigate to `ContentSingle` or `ContentSeriesSingle`
2. see that columns are divided to 2 max

## 📸 Screenshots


| Before | After |
| --- | --- |
| <img width="1349" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/68402088/e68275e8-24ee-4411-be5e-0c1781eff9d4">| <img width="1333" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/68402088/cfb5f1a4-46cb-475c-8d00-da29051802e5"> |
